documentation/source/conf.py

```diff
@@ -132,11 +132,15 @@
 }
 
 html_context = {
-    "display_github": True,
-    "github_user": "neuropoly",
-    "github_repo": "spinalcordtoolbox",
-    "github_version": "master",
-    "conf_py_path": "/documentations/",
+    # TODO: when the Github icon is supported natively by furo (https://github.com/pradyunsg/furo/discussions/114)
+    # then this should be moved into html_theme_options and the theme_ prefix should be dropped
+    "theme_source_repository": "https://github.com/spinalcordtoolbox/spinalcordtoolbox",
+    "theme_source_branch": "master",  # or subprocess.check_output(["git", "rev-parse", "--abbrev-ref", "HEAD"]) ?
+    "theme_source_directory": "documentation/source/",
+
+    # TODO: this should be determined automatically, but it seems that *assigning* to html_context wipes out
+    # the automatically determined value?
+    "page_source_suffix": "rst",
```

Comment:
This is super odd! Like your TODO comment says, this should be [determined automatically](https://www.sphinx-doc.org/en/master/templating.html#page_source_suffix):

> In documents that are created from source files (as opposed to automatically-generated files like the module index, or documents that already are in HTML form), these variables are also available:
> 
> [...]
>
> `page_source_suffix`
>    The suffix of the file that was rendered. Since we support a list of `source_suffix`, this will allow you to properly link to the original source file.

:thinking:

Reply:
This could have adverse consequences if we ever branch out to Markdown files in SCT, but... I imagine this is fine to just hardcode...

Still curious though.

Reply:
The edit button really be fixed in `furo` itself (they're working on it, they just haven't done it yet), and when that happens we can drop `html_context` and the problem should go away.